Leaflet.Graticule.js

```diff
@@ -187,43 +189,102 @@ L.LatLngGraticule = L.Layer.extend({
         L.DomUtil.setOpacity(this._canvas, this.options.opacity);
     },
 
+    // todo: Decimal degrees to Degrees - Decimal Minutes
+    __deg_to_dmd: function(deg) {
+
+        var d = Math.floor (deg);
+        var minfloat = (deg - d) * 60;
+        var min = minfloat.toFixed(1);
+
+        if (min == 60.0) {
+            d ++;
+            min = 0.0;
+        }
+
+        if (min == 0.0){
+            return ('' + d + '°');
+        }
+
+        if ((min % 1).toFixed(1) == 0.0) {
+            min = minfloat.toFixed(0).toString(10).padStart(2, '0');
+        }
+        else {
+            min = minfloat.toFixed(1).toString(10).padStart(4, '0');
+        }
+        
+        return ('' + d + '°' + min + "'");
+    },
+
     __format_lat: function(lat) {
         if (this.options.latFormatTickLabel) {
             return this.options.latFormatTickLabel(lat);
         }
 
-        // todo: format type of float
-        if (lat < 0) {
-            return '' + (lat*-1) + 'S';
+        if (this.options.dmd) {
+            if (lat < 0) {
+                return this.__deg_to_dmd(lat * -1) + ' S';
+            }
+            else if (lat > 0) {
+                return this.__deg_to_dmd(lat) + ' N';
+            }
+            return this.__deg_to_dmd(lat);
         }
-        else if (lat > 0) {
-            return '' + lat + 'N';
+        else {
+            // todo: format type of float
+            if (lat < 0) {
+                return '' + (lat*-1).toFixed(this.options.precision) + 'S';
+            }
+            else if (lat > 0) {
+                return '' + lat.toFixed(this.options.precision) + 'N';
+            }
+            return '' + lat.toFixed(this.options.precision);
+
         }
-        return '' + lat;
     },
 
     __format_lng: function(lng) {
         if (this.options.lngFormatTickLabel) {
             return this.options.lngFormatTickLabel(lng);
         }
 
-        // todo: format type of float
-        if (lng > 180) {
-            return '' + (360 - lng) + 'W';
-        }
-        else if (lng > 0 && lng < 180) {
-            return '' + lng + 'E';
-        }
-        else if (lng < 0 && lng > -180) {
-            return '' + (lng*-1) + 'W';
-        }
-        else if (lng == -180) {
-            return '' + (lng*-1);
+        if (this.options.dmd) {
+            // todo: format type of float
+            if (lng > 180) {
+                return this.__deg_to_dmd(360 - lng) + ' W';
```

Comment:
Use modulus 360 (i.e. `lng % 360`) instead.

Or better yet, use `wrapLatLng()` of the map's CRS. (i.e `this._map.options.crs.warpLatLng(latlng)`) on the appropriate place, or even `this._map.options.crs.warpLatLng([0, lng]).lng`. That should care of maps with non-default CRSs.